### PR TITLE
Update CI for TIMM models

### PR DESCRIPTION
**Pull request by desertfire**
Summary:
1) Relax the exception check for retry when pretrain model download fails; 
2) Enable accuracy checking for inductor training as we are over 90% passing on TIMM

### CODE REVIEW

**Review by anijain2305 — APPROVED** [review]

**Comment by anijain2305** [comment]
> 1. Relax the exception check for retry when pretrain model download fails;

Maybe the right way to handle this is caching the pretrained weights in CI?

**Comment by desertfire** [comment]
> > 1. Relax the exception check for retry when pretrain model download fails;
> 
> Maybe the right way to handle this is caching the pretrained weights in CI?

Agree, but I don't know how to do it.

Pretrained models are downloaded to directories like `/home/circleci/.cache/torch/hub/checkpoints/gluon_inception_v3-9f746940.pth`, which is not under the project directory, `/home/circleci/project`. Is there an easy way to config where to download TIMM models?
